src/converters/RpcHttpConverters.ts

```diff
@@ -62,7 +62,10 @@ export function toRpcHttp(inputMessage): rpc.ITypedData {
     const httpMessage: rpc.IRpcHttp = inputMessage;
     httpMessage.headers = toRpcHttpHeaders(inputMessage.headers);
     httpMessage.cookies = toRpcHttpCookieList(inputMessage.cookies || []);
-    const status = inputMessage.statusCode || inputMessage.status;
+    let status = inputMessage.statusCode;
```

Comment:
The is the only source code I changed in this PR. Basically, my new tests revealed that it was possible for us to stringify the `inputMessage.status` _method_ on `Response` getting this literal value: 

`"status(statusCode) {\n        this.statusCode = statusCode;\n        return this;\n    }"`

Reply:
Wow, good catch! I guess another argument against overloading the `status` property

Reply:
Does this affect being able to set the response code using 

```TS 
context.res = {
    status: 200
}
```
though?

Reply:
I just tested this.

```TS
context.res = {
    status: 404,
};
```

This would return a 404 status code with the old code, but return 200 with this change.

Reply:
OOoops sorry ignore me I just noticed the rest of your change 🤦🏽‍♂️